boards/nordic/nrf5340_audio_dk/nrf5340_audio_dk_nrf5340_cpuapp_common.dtsi

```diff
@@ -137,23 +137,6 @@ arduino_i2c: &i2c1 {
 	pinctrl-0 = <&i2c1_default>;
 	pinctrl-1 = <&i2c1_sleep>;
 	pinctrl-names = "default", "sleep";
-};
-
-arduino_spi: &spi4 {
```

Comment:
You should correct the commits. Currently, the second commit removes changes in `boards/` that are just unnecessarily introduced by the first one.